client/me/concierge/shared/closure-notice.js

```diff
@@ -34,7 +34,7 @@ const ClosureNotice = ( {
 
 	if ( currentDate.isBefore( i18n.moment.utc( closureStartDate ) ) ) {
 		message = translate(
-			'{{strong}}Note:{{/strong}} Concierge will be closed %(closureStartDate)s through ' +
+			'{{strong}}Note:{{/strong}} Support session will be closed %(closureStartDate)s through ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( '{{strong}}Note:{{/strong}} Concierge will be closed %(closureStartDate)s through %(closureEndDate)s for the%(holidayName)s holiday. If you need to get in touch with us, you’ll be able to {{link}}submit a support request{{/link}} and we’ll get to it as fast as we can.' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).